homeassistant/components/seventeentrack/sensor.py

```diff
@@ -157,7 +162,9 @@ def _async_create_remove_entities():
         for status, summary_data in coordinator.data.summary.items()
     )
 
-    _async_create_remove_entities()
+    if not config_entry.data.get("deprecated"):
+        deprecate_sensor_issue(hass, config_entry.entry_id)
+        _async_create_remove_entities()
 
     config_entry.async_on_unload(
         coordinator.async_add_listener(_async_create_remove_entities)
```

Comment:
Should we still register this listener when its deprecated?

Reply:
moved